### PR TITLE
Admin - Improve page performance

### LINKED ISSUE #4976: Roles missing in 'Role mapping by VLAN ID' while editing switches or switch groups (pf 9.0.1)
**Describe the bug**
After adding a new role at 'Configuration -> Roles', most of the defined roles are missing from 'Configuration -> Policies and Access Control -> Network Devices -> Switches (or Switch Groups)' (see attached screenshot... I have about 250 defined roles... only the first 28 appear in the table...)

**To Reproduce**
1. Go to 'Configuration -> Policies and Access Control -> Network Devices -> Switches (or Switch Groups)' 
2. Select a switch or switch group
3. Click on 'Roles'
4. See (or better, don't see...) all your defined roles

**Screenshots**
Roles at selected switch group
Roles at selected switch group

All defined roles... about 250 roles
All defined roles... about 250 roles

**Expected behavior**
Seeing all defined roles and being able to work with them...

**Desktop (please complete the following information):**
 - OS: Debian Buster (10.2)
 - Browser: Chrome, Firefox, ... doesn't matter

### PR DESCRIPTION
**Pull request by satkunas**
# Description
Refactor vuelidate/form models into a single store module in order to improve VueJS DOM reflow performance with complex forms.

# Impacts
* Sections: Configuration, Nodes, Users
* VueJS build needed
* No new packages

# Issue
fixes #4976 

# Delete branch after merge
YES

# Checklist
(REQUIRED) - [yes, no or n/a]
- [n/a] Document the feature
- [n/a] Add unit tests
- [n/a] Add acceptance tests (TestLink)

## Enhancements
* Improved Admin performance, refactor form/validation model on page views in Configuration, Nodes and Users section. 
* Tidied up configuration exports. Reduced verbosity in naming. Simplified validators.
* Various form input component improvements. Simplified components.

## Bug Fixes
* Editing switches, users, auth sources, conn. profiles is extremely slow in new admin (#4697)

### CODE REVIEW

File: html/pfappserver/root/static.alt/src/views/Configuration/_components/AdminRoleView.vue [hunk 1]
```diff
@@ -79,30 +74,21 @@ export default {
...
-      return this.$v.form.$invalid || this.$store.getters[`${this.storeName}/isWaiting`]
+      return this.$store.getters[`${this.formStoreName}/$formInvalid`]
     },
-    getForm () {
-      return {
-        labelCols: 3,
-        fields: fields(this)
-      }
+    isLoading () {
+      return this.$store.getters['$_admin_roles/isLoading']
```
Comment (cgx) [inline html/pfappserver/root/static.alt/src/views/Configuration/_components/AdminRoleView.vue]: Should it be `this.formStoreName` instead?

**Comment by lzammit** [comment]
User creation: I can set a date past 2038-01-019 00:00:00

You should show an error that says that the unregistration windows can't exceed that date.

**Comment by lzammit** [comment]
User creation:  I have to put the start date for the Registration Window
before it auto field with today's date

**Comment by lzammit** [comment]
User creation: No gender in the drop down list:

image

**Comment by lzammit** [comment]
Switch creation/edition: Role fields are a bit slow.

**Comment by lzammit** [comment]
Connection profile: Can't add new data in the filter field.

If I want to add an SSID name that no device connect to:

Before:

image

Now: 
image

**Comment by lzammit** [comment]
Connection profile: Language not selected by default:

image

**Comment by lzammit** [comment]
Sources: Can't create an AD source:

image

image

Source config:

image

**Comment by extrafu** [comment]
That PR was just too small to be merged!

**Comment by satkunas** [comment]
`+` should have been smaller
 `-` should have been bigger

**Comment by extrafu** [comment]
o should have been taller

**Pull request merged**